From a15187b6f31af4cfae328ffdd6b8c3baf559e755 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Tue, 4 Apr 2023 23:32:38 +0300
Subject: [PATCH] rpi4: Remove preboot commands

With recent uboot it causes "Unexpected XHCI event TRB, skipping"
error and continuous reboot.

Also it affects boot time.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 configs/rpi_4_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/rpi_4_defconfig b/configs/rpi_4_defconfig
index a6fe2feab2..5008ee3509 100644
--- a/configs/rpi_4_defconfig
+++ b/configs/rpi_4_defconfig
@@ -11,7 +11,7 @@ CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0x7fffe30
 CONFIG_OF_BOARD_SETUP=y
 CONFIG_USE_PREBOOT=y
-CONFIG_PREBOOT="pci enum; usb start;"
+CONFIG_PREBOOT=""
 # CONFIG_DISPLAY_CPUINFO is not set
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_MISC_INIT_R=y
-- 
2.37.2

